zypper-migration

```diff
@@ -106,6 +106,31 @@ def check_if_param(opt, message)
   end
 end
 
+def get_available_services_based_on_url
+  url = SUSE::Connect::Config.new().url
+  url = "suse.com" if url.nil?
```

Comment:
`SUSE::Connect::Config.url` should never be `nil`. It defaults to `https://scc.suse.com` https://github.com/SUSE/connect/blob/master/lib/suse/connect/config.rb#L28.